gatsby/searchtraversal.rst

```diff
@@ -0,0 +1,147 @@
+Search Traversal Method Of Retrieving Data
+==========================================
+
+In the previous chapter we covered how to fetch data for a single content object.
+For our source-plugin we will need to fetch the data for all the content objects in a Plone site and process it into nodes.
+
+One of the strategies that we experimented with and adopted for the source-plugin was "Search Traversal".
+
+
+Getting The Full List Of Content
+--------------------------------
+
+Make a GET Request to `https://plonedemo.kitconcept.com/en/@search`:
+
+.. code-block:: json
+  {
+    "@id": "https://plonedemo.kitconcept.com/en/@search",
+    "items": [
+        {
+            "@id": "https://plonedemo.kitconcept.com/en",
+            "@type": "LRF",
+            "description": "",
+            "review_state": "published",
+            "title": "English"
+        },
+        {
+            "@id": "https://plonedemo.kitconcept.com/en/media",
+            "@type": "LIF",
+            "description": "",
+            "review_state": "published",
+            "title": "Media"
+        },
+        {
+            "@id": "https://plonedemo.kitconcept.com/en/frontpage",
+            "@type": "Document",
+            "description": "The ultimate Open Source Enterprise CMS",
+            "review_state": "published",
+            "title": "Welcome to Plone 5"
+        },
+   ...
+
+
+We see that the ``@search`` endpoint at the Plone root returns a flat list of all content objects in the site.
```

Comment:
Maybe it will be a good idea to provide some postman example calls and replies.
You say "we see" but we are not seeing anything ;)

@stevepiercy you are more "english" than me: this phrase doesn't sounds very well to me..is it better to write something like the following (more or less)?

With @search call we could see a flat list of results like this:
{EXAMPLE}

There are only few information here. To build a node, we need more data, so we could try to call the @id endpoint and see what does it returns:
{EXAMPLE}

that's exactly what we need.

Now we need to combine these two calls to get full data from every site contents:

{EXAMPLE}